dist/mimedown.spec

```diff
@@ -0,0 +1,41 @@
+# run "spectool -g -R mimedown.spec" to automatically download source files
+# spectool is part of the rpmdevtools package
+%global commit 2f298c3e53ad49e6c2aa0e1cca940ab04fa7f02c
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+%global repo begriffs
+
+Name:       mimedown
+Version:    2f298c3
+Release:    1%{?dist}
+Summary:    A tool to display user-oriented Slurm partition information.
+
+Group:      System Environment/Base
+License:    GPLv2+
+URL:        https://github.com/%{repo}/%{name}
+Source:     https://github.com/%{repo}/%{name}/archive/%{commit}/%{name}-%{shortcommit}.tar.gz
+
+BuildRequires:   cmark-devel
```

Comment:
Should we also list a C compiler and pkg-config in BuildRequires? (Furthermore, is there a way to specify that *any* compiler will do, rather than picking gcc or clang in particular?)

Reply:
Actually, yes, it's a good idea and library dependencies can be [specified as `pkgconfig()`](https://docs.fedoraproject.org/en-US/packaging-guidelines/PkgConfigBuildRequires/)

Now, for the compiler, I'm not sure. i don't think there is a generic "C compiler" package tag, and it would probably not be consistent across RPM-based distributions anyway. But dependencies can be expressed as [boolean expressions](https://rpm.org/user_doc/boolean_dependencies.html), so I guess we could have `(gcc or clang)`?

Reply:
Ah but boolean dependencies are supported "Starting with rpm-4.13". RHEL/CentOS 7 still use version 4.11, so that won't be very portable.